pkg/resources/management.cattle.io/v3/fleetworkspace/validator.go

```diff
@@ -19,6 +21,8 @@ import (
 	"k8s.io/kubernetes/pkg/registry/rbac/validation"
 )
 
+const fleetManagedLabel = "fleet-workspace-managed-resource"
```

Comment:
I would suggest using a standard label like `app.kubernetes.io/managed-by: rancher`
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
done in https://github.com/rancher/webhook/pull/260/commits/24334874f8922d5187275b7fdf0f1c85dfcacf5f